content/pages/sponsorship.md

```diff
@@ -0,0 +1,157 @@
+In 2022, we assembled an [all-volunteer team](https://a11ytalks.com/pages/about), passionately contributing time and energy to sustain the project, and we became fiscally hosted by a non-profit, Open Collective Foundation (501)(c)(3).
+
+Now, we need your support to help us meet our accessibility goals, compensate our amazing speakers, and improve our live stream productions to provide the best value possible for our community.
+
+## Budget Priorities for 2023
+
+1. __Live captions__ ($150/month, $1,800/annualy): Staying true to our value of accessibility, high quality live captions are our top priority to ensure our live stream is accessible to as many people as possible. Our current vendor provides live human-generated captions.
+1. __Speaker stipend__ ($100/month, $1,200/annually): We believe in compensating our speakers for the time they spend creating amazing content and sharing it with our viewers.
+1. __Streaming service__ ($20/month, $240/annually):
+Streamyard is our tool of choice. A paid subscription gives us more flexibility and control in how our talks are presented, giving us all of the [features of the basic plan](https://streamyard.com/pricing).
+1. __Live American Sign Language interpreter__ ($300/month, $3,600/annually):
+Including live <abbr title="American Sign Language">ASL</abbr> interpretation would allow us to reach even more people than captions and further increase the accessibility of our content. See [Shattering Assumptions About Deafness](https://youtu.be/hTZqC6uVmuQ) - Meryl Evans for an example of our content delivered in ASL for viewers.
+1. __Operations__ ($54/month, $645/annually): Operations include necessary technical items such as domain renewal, web hosting, etc. We have also included general liability insurance, which would protect the A11yTalks project from claims like libel or slander. *These costs are billed annually.*
+
+## How to give
+
+Your financial contributions are tax deductible because A11yTalks is fiscally hosted by a non-profit, ​Open Collective Foundation 501(c)(3).
+
+### Individuals: Join the A11yCat Club
+
+You can set-up a [Monthly Recurring Donation](https://opencollective.com//a11ytalks/contribute/a11ycat-club-36522) or a [One-time Donation](https://opencollective.com//a11ytalks/contribute/one-time-donation-47987) through our Open Collective page. Each level has some contributor benefits as our way of saying “Thanks for being a part of A11yTalks!”
+To receive your A11yCat T-shirt and/or sticker, please follow the instructions on your donation confirmation email from Open Collective. These will be mailed out in monthly batches. Email us at “team [at] a11ytalks [dot] com” if you think we’ve missed you. We promise it wasn’t on purpose.
+
+<table class="centered">
+<thead class="inverse">
+<tr>
+ <th colspan="4">Join the A11yCat Club (Monthly Recurring)</th>
```

Comment:
Make this a link to https://opencollective.com//a11ytalks/contribute/a11ycat-club-36522.